<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
	<class>connectDialog</class>
	<widget class="QDialog" name="connectDialog">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>180</width>
				<height>130</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Dialog</string>
		</property>
		<widget class="QLabel" name="label">
			<property name="geometry">
				<rect>
					<x>10</x>
					<y>10</y>
					<width>50</width>
					<height>20</height>
				</rect>
			</property>
			<property name="text">
				<string>服务器IP</string>
			</property>
			<property name="alignment">
				<set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
			</property>
		</widget>
		<widget class="QLabel" name="label_2">
			<property name="geometry">
				<rect>
					<x>10</x>
					<y>40</y>
					<width>50</width>
					<height>20</height>
				</rect>
			</property>
			<property name="text">
				<string>端口</string>
			</property>
			<property name="alignment">
				<set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
			</property>
		</widget>
		<widget class="QLineEdit" name="lineEdit">
			<property name="geometry">
				<rect>
					<x>70</x>
					<y>10</y>
					<width>100</width>
					<height>20</height>
				</rect>
			</property>
		</widget>
		<widget class="QLineEdit" name="lineEdit_2">
			<property name="geometry">
				<rect>
					<x>70</x>
					<y>40</y>
					<width>100</width>
					<height>22</height>
				</rect>
			</property>
		</widget>
		<widget class="QPushButton" name="pushButton">
			<property name="geometry">
				<rect>
					<x>20</x>
					<y>80</y>
					<width>140</width>
					<height>35</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<family>Microsoft YaHei</family>
					<pointsize>15</pointsize>
				</font>
			</property>
			<property name="text">
				<string>连接</string>
			</property>
		</widget>
	</widget>
	<resources/>
	<connections/>
</ui>
